### PR TITLE
Move CM_Util::varDump() to CM_Debug_VariableInspector

### PR DESCRIPTION
**Pull request by njam**
E.g.:
```php
$variableInspector = new CM_Debug_VariableInspector();
$variableInspector->getDebugInfo($variable);
```

@tomaszdurka thoughts?

### CODE REVIEW

**Comment by njam** [comment]
@tomaszdurka @mariansollmann please review

File: tests/library/CM/Debug/VariableInspectorTest.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            ['SplFixedArray', new SplFixedArray()],
+        ];
+    }
+
+    public function testGetDebugInfoLengthMax() {
+        $variableInspector = new CM_Debug_VariableInspector();
+        $this->assertSame("'foo...'", $variableInspector->getDebugInfo('fooo', ['lengthMax' => 3]));
+        $this->assertSame("'fooo'", $variableInspector->getDebugInfo('fooo', ['lengthMax' => 4]));
+        $this->assertSame("'fooo'", $variableInspector->getDebugInfo('fooo', ['lengthMax' => null]));
+    }
```
Comment (mariansollmann) [inline tests/library/CM/Debug/VariableInspectorTest.php]: maybe add a test for `lengthMax` more than actual chars in string?

**Comment by mariansollmann** [comment]
rest looks good

**Comment by tomaszdurka** [comment]
Looks good

**Pull request merged**